packages/dx-react-grid/docs/guides/plugin-overview.md

```diff
@@ -9,7 +9,7 @@ The plugins that provide all the Grid's functionality can be divided into four g
 
 Note that the plugins are composable and can be nested into each other.
 
-Refer to the [Reference](../../reference/grid.md) to see the complete plugin list.
+Refer to the [Reference](../../reference/grid) to see the complete plugin list.
```

Comment:
`*.md ` is OK. The problem is the 'reference' folder path is incorrect here. `../../reference/` should be replaced with `../reference/`. Well, the correct link is `../reference/grid.md`